test/integration/integration_test_utils.py

```diff
@@ -14,60 +14,31 @@
 decision making for integration tests."""
 import os
 
-import botocore.session
-from six.moves.configparser import ConfigParser
 from aws_encryption_sdk.key_providers.kms import KMSMasterKeyProvider
 
-SKIP_MESSAGE = 'Skipping tests due to blocking environment variable'
+SKIP_MESSAGE = (
+    'Required environment variables not found. Skipping integration tests.'
+    ' See integration tests README.rst for more information.'
+)
+TEST_CONTROL = 'AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_CONTROL'
+AWS_KMS_KEY_ID = 'AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID'
 
 
 def skip_tests():
-    blocker_var_name = 'AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_CONTROL'
-    blocker_val = os.environ.get(blocker_var_name, None)
-    if blocker_val != 'RUN':
-        return True
-    return False
+    """Only run tests if both required environment variables are found."""
+    test_control = os.environ.get(TEST_CONTROL, None)
+    key_id = os.environ.get(AWS_KMS_KEY_ID, None)
+    return not (test_control == 'RUN' and key_id is not None)
```

Comment:
Minor: not a huge deal given the context, but could be a bit more readable for maintenance's sake; even just expanding the negate out would help: `test_control != 'RUN' or key_id is None` (if I did that right)

Reply:
I'll be refactoring a decent chunk of the integration test utils when we get the test vectors packaged as a Python package, so I'll leave refining this bit until then.